roles/common/files/letsencrypt-gencert

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+service apache2 stop
```

Comment:
I am thinking about the invariant.  Apache is running when this task list runs.  After the gencert script (which is run conditionally), Apache is left running.  I don't see the need for the last start task, and I worry it will confuse future readers who also wonder why it's there, trying to answer the question, "what is the condition that led to the author thinking they needed to (re)start Apache from Ansible???"